README.md

```diff
@@ -283,7 +283,9 @@ And finally, you are ready to go! Or at least you should be...
 
 ### Junior Contributors
 
+**Jaume Jurado** | [Contact me][JJS1440] | Phrase bla bla bla...
```

Comment:
Istead of  "[Contact me][JJS1440] "   use [JJS1440][JJS1440]  this way we can see your username.